src/components/messages_window/messages_window.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './messages_window.scss';
+
+import { Button, Form } from 'react-bootstrap';
+import { lang } from '../../lang/EN';
+
+import { Message } from '../message/message';
+import { MessageType } from '../../types/message';
+
+export type MessageWindowProps = {
+  messages: MessageType[];
+};
+
+export const MessagesWindow = ({
+  messages
+}: MessageWindowProps): React.ReactElement => {
+  return (
+    <div className="messages-window">
+      <div className="messages-window__window">
+        {messages
+          ? messages.map((message, index) => (
```

Comment:
Please, don't use index as a key.  Add field `id` to `MessageType`

Reply:
TODO added